README.md

```diff
@@ -64,6 +64,12 @@ to the controller-gen CLI page in the [kubebuilder documentation](https://book.k
 
 ### Running the operator in kind
 
+#### Automated
+
+Run `./hack/kind/setup.sh` to create a kind cluster that has all the
+required dependencies (OLM, infra-node, registry).
```

Comment:
Do you think it also makes sense to add a teardown script? 🤔 
If not I feel that at least deprovision instructions would be nice
```
kind delete cluster --name obs-operator
```

Reply:
The script does print how to delete the cluster but no harm in mentioning it here. We do have to keep the docs updated if anything changes in the script.